### DISCUSSION TITLE
Bluetooth access?

### DISCUSSION THREAD

**Discussion by luisrieke** [responsively-org/responsively-app#704] [General]
Why does the app asks for bluetooth access?

**Comment by jjavierdguezas** [comment]
Hi @luis-rieke how can we test this? Could you provide more info, please?

**Reply by luisrieke** [reply]
I operate on a MacBook Pro, i7, with macOS BigSur - what else do you need?
When I first started the App it asked for Bluetooth Access. I refused. No problems since then, everything works. But was wondering why this was requested, seemed a bit strange to me to be honest.. I'm still testing whether the tool makes sense for our purpose.

**Reply by jjavierdguezas** [reply]
@manojVivek have you seen this behavior on Mac? I dont see it on Windows...

**Reply by manojVivek** [reply]
I haven't seen the behaviour as well. Might be from the page getting loaded? 🤔 

@luis-rieke Can you please share a screenshot of the permission request prompt?

**Reply by jjavierdguezas** [reply]
@luis-rieke responsively does not need bluetooth to operate so this is very unusual. Could it be that the website being loaded asked for that permission? like this:

![image](https://user-images.githubusercontent.com/13673443/177291544-c38d85d5-86d9-4fba-8fa9-cddd48700049.png)

**Reply by luisrieke** [reply]
Unfortunately I can't reproduce it anymore. Maybe it really was the opened webpage..
